image_geometry/include/image_geometry/pinhole_camera_model.h

```diff
@@ -1,10 +1,11 @@
 #ifndef IMAGE_GEOMETRY_PINHOLE_CAMERA_MODEL_H
 #define IMAGE_GEOMETRY_PINHOLE_CAMERA_MODEL_H
 
-#include <sensor_msgs/CameraInfo.h>
+#include <sensor_msgs/msg/camera_info.hpp>
 #include <opencv2/core/core.hpp>
 #include <opencv2/imgproc/imgproc.hpp>
 #include <opencv2/calib3d/calib3d.hpp>
+#include <boost/shared_ptr.hpp>
```

Comment:
can you use `std::shared_ptr` given that that's what is used everywhere in the codebase? seems overkill for this package to depend on boost only for shared pointers